llvm/lib/Support/Z3Solver.cpp

```diff
@@ -50,16 +45,17 @@ void Z3ErrorHandler(Z3_context Context, Z3_error_code Error) {
 /// Wrapper for Z3 context
 class Z3Context {
```

Comment:
Another class that manages a resource in a destructor, but does not handle copy/move assignment/construction.